packages/devtools-reps/src/object-inspector/component.js

```diff
@@ -155,7 +156,7 @@ class ObjectInspector extends Component {
   }
 
   setExpanded(item: Node, expand: boolean) {
-    if (nodeIsPrimitive(item)) {
+    if (nodeIsPrimitive(item) && !nodeIsLongString(item)) {
```

Comment:
I wonder if we should return false in nodeIsPrimitive for long strings 🤔 
It would make sense since now we don't render top-level primitives using the launchpad.
But I don't know if there would be unwanted side effects doing that

Reply:
Omitting longstrings from primitives would influence:
- `shouldRenderRootsInReps`, making longStrings not rendered as reps directly, which is what we want since we want them within ObjectInspector:
  ```javascript
  if (shouldRenderRootsInReps(roots)) {
    return renderRep(roots[0], props);
  }
  return new OI(props);
  ```
- `shouldLoadItemPrototype`, loading prototypes for them
- `shouldLoadItemSymbols`, loading symbols for them

Reply:
If I understand correctly, we don't want any symbols and prototype for it but we want it in the OI.

In the OI it shouldn't have any children and in `renderTreeItem` we probably need to add a `renderTreeItemValue` that renders if roots is longString, since we don't need labels for them.

Lastly `setExpanded` should should toggle value.member.open if it is root and longString.

Reply:
Thanks for digging into that @kootoopas ! So, looks like we should keep longString a part of nodeIsPrimitive and add if statements in the places that need so.

Reply:
Hmm, 🤔  maybe we shouldn't. We can add checks to skip prototype, symbols or whatever unwanted loadable property for a longString while treating it as a non primitive.